internal/mode/static/state/graph/service_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"testing"
 
 	. "github.com/onsi/gomega"
+
```

Comment:
These should stay grouped

Reply:
hmm, I took it out because I think it added it when I had expanded on the service tests and since I removed all my additional service test changes I saw that there was a space missing here and that was the only additional change in the file, was this just not caught in the linter and a previous PR?